apps/meeting/src/components/Spinner/index.tsx

```diff
@@ -0,0 +1,84 @@
+// Copyright 2020-2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import React from 'react';
+import { StyledSpinner } from './Styled';
+
+const Spinner: React.FC = () => {
+  return (
+    <StyledSpinner>
+      <svg
+      xmlns="http://www.w3.org/2000/svg"
+      viewBox="0 0 24 24"
+      className='spinner'
+      >
+        <g fill="none" fillRule="evenodd">
+          <path d="M0 0H24V24H0z" />
+          <g fill="currentColor">
+            <path
```

Comment:
Instead of adding all the path components individually it would be better to keep the values as array of object and we can map the array to generate all the path components.

Reply:
I am not actually familiar with svg so would need more details. This I just copied from the Chime App :)